conf.json

```diff
@@ -986,6 +987,16 @@
 				"def": "true",
 				"descr": "Should the client adapt metachunk size to EC policy parameters? Letting this on will make bigger metachunks, but chunks on storage will stay at normal chunk size. Disabling this option allows clients to do write alignment." },
 
+			{ "type": "int64", "name": "oio_chunk_size_minimum",
+				"key": "core.chunk_size.minimum",
```

Comment:
Cosmetic, minor: other variable names use "min" and "max" words instead of "minimum" and "maximum". For example `client.errors_cache.max` and `sqliterepo.repo.fd_min_active`.

Reply:
I agree. Let's have uniform names and aliases to manage existing productions.